cfgov/ask_cfpb/wagtail_hooks.py

```diff
@@ -111,3 +100,37 @@ def register_export_menu_item():
 @hooks.register('register_admin_urls')
 def register_export_url():
     return [url('export-ask', export_data, name='export-ask')]
+
+
+def get_feedback_stream_value(page):
+    """Delivers a basic feedback module with yes/no buttons and comment box"""
+    translation_text = {
+        'helpful': {'es': '¿Fue útil esta respuesta?',
+                    'en': 'Was this page helpful to you?'},
+        'button': {'es': 'Enviar',
+                   'en': 'Submit'}
+    }
+    stream_value = [
+        {'type': 'feedback',
+         'value': {
+             'was_it_helpful_text': translation_text['helpful'][page.language],
+             'button_text': translation_text['button'][page.language],
+             'intro_text': '',
+             'question_text': '',
+             'radio_intro': '',
+             'radio_text': ('This information helps us '
+                            'understand your question better.'),
+             'radio_question_1': 'How soon do you expect to buy a home?',
+             'radio_question_2': 'Do you currently own a home?',
+             'contact_advisory': ''}}]
+    return stream_value
+
+
+@hooks.register('after_create_page')
+def after_create_page(request, page):
+    if isinstance(page, AnswerPage):
+        page.content = StreamValue(
+            page.content.stream_block,
+            get_feedback_stream_value(page),
+            is_lazy=True)
+        page.save_revision(user=request.user)
```

Comment:
This is very minor since it's unlikely to happen often, but I was wondering if we should publish the revision if the user is publishing the page? Otherwise the page's status would be live + draft rather than live, which might be confusing for the user.